src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -61,21 +61,28 @@ public function __construct(LoaderInterface $loader = null)
      */
     public function import($resource, $prefix = '/', $type = null)
```

Comment:
in case of a glob, this is missing tracking of the directory prefixing the pattern - see "`$this->container->fileExists($resourcePrefix, '/^$/');`" in DI.
Should we do it inline here? Would be like a leak but do we have a better choice?